hoot-core-test/src/test/cpp/hoot/core/io/ServicesDbTestUtils.cpp

```diff
@@ -47,9 +47,11 @@
 
 #include "../TestUtils.h"
 
+using namespace std;
+using namespace Tgs;
+
 namespace hoot
 {
-using namespace Tgs;
```

Comment:
Moved this up out of the `hoot` namespace and put it with the `using namespace std` directive.  This happens a lot.